content/hi/kubernetes.md

```diff
@@ -0,0 +1,24 @@
+---
+title: कुबेरनेट्स (Kubernetes)
+status: completed
+category: अवधारणा
+exclude_search: true

```

Comment:
We have to include a tags column instead of exclude_search. Please add that.
Ref: https://github.com/cncf/glossary/blob/main/content/en/kubernetes.md

Reply:
@devsachinsharma are you still working on this?